axe_selenium_python/tests/test_axe.py

```diff
@@ -4,45 +4,51 @@
 
 from os import path
 import pytest
```

Comment:
You should follow [PEP8](https://www.python.org/dev/peps/pep-0008/#imports) for import order.